packages/app/src/Application.ts

```diff
@@ -45,7 +45,7 @@ export interface Application extends GlobalMixins.Application {}
  * @class
  * @memberof PIXI
  */
-export class Application
+export class Application<VIEW = ICanvas>
```

Comment:
Just realised, this should be `VIEW extends ICanvas = ICanvas` otherwise we would be able to create an Application with anything (e.g. `new Application<string>()`), but then you get the following error when creating an Application with `new Application<NodeCanvasElement>()`:

```
Type 'NodeCanvasElement' does not satisfy the constraint 'ICanvas'.
  The types returned by 'getContext(...)' are incompatible between these types.
    Type 'CanvasRenderingContext2D | WebGLRenderingContext' is not assignable to type 'ICanvasRenderingContext2D'.
```

Reply:
Oh, the `NodeCanvasElement` problem is there since I made interface `ICanvas`. The `NodeCanvasElement` has some `// @ts-expect-error` in its current implementation (which is bad and unnecessary in my opinion), so maybe we need some breaking changes to make `NodeCanvasElement` implicit convertible to `ICanvas`.

Reply:
See PR #8823.